utility.py

```diff
@@ -1,6 +1,35 @@
 from datetime import date, datetime
 import psycopg2
+import requests
+from typing import List
 import os
+import argparse
+from bs4 import BeautifulSoup
+from opportunity import Opportunity
```

Comment:
> For some reason, in `gpt_utils.py`, whenever I tried to do `from opportunity import Opportunity` I recieved this error.
> 
> ```lisp
> File "C:\Users\boush\OneDrive\Documents\GitHub\acmOpportunities\opportunity.py", line 4, in <module>
>     import utility as utils
>   File "C:\Users\boush\OneDrive\Documents\GitHub\acmOpportunities\utility.py", line 8, in <module>
>     from opportunity import Opportunity
> ImportError: cannot import name 'Opportunity' from partially initialized module 'opportunity' (most likely due to a circular import) (C:\Users\boush\OneDrive\Documents\GitHub\acmOpportunities\opportunity.py)
> ```
> 
> If anyone has any ideas, please let me know! I tried to figure it out, but I can't quite get to the reason why I was recieving this. I removed the import for now.

You may either import `opportunity.py` from `utility.py` or you may import `utility.py` from `opportunity.py`. But you may not do both.

Reply:
Fixed [here](https://github.com/acmcsufoss/acmOpportunities/pull/11/commits/fdeeb4acca854730a8315bcd1594017d233b4043). 

Ended up deleting `gpt_utils.py` as it had no use and those functions can be easily integrated in `utility.py`.